changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Facebook for WooCommerce Changelog ***
 
 2020.nn.nn - version 2.1.2
+ * Tweak - Ensure default variation is carried over to Facebook

```

Comment:
Since the product item ID of the default variation is not available on initial sync (because each variation will be synced asynchronously using the Batch API), I think we should update the changelog entry to indicate when can merchants expect to see the default variation reflected on their catalogs.

Perhaps something like this (should update `readme.txt` as well):

```suggestion
 * Tweak - Default variation selection will be synced to Facebook if the default product variation is already synced 

```